content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -59,14 +62,17 @@ In this situation, the client and server will simultaneously broadcast messages
 A `server` socket is a subclass of the ServerSocket that uses the constructor below:
 
 ```java
-serversocket = new ServerSocket (int port)
+serversocket = new ServerSocket(int port)
 ```
 Port: the port number on which the server will listen for the client's requests.
 
-The ServerSocket object stays idle, waiting for network requests. Our client will attempt to connect to port 5000, which we have specified.
+>The ServerSocket object stays idle, waiting for network requests. Our client will attempt to connect to port 5000, which we have specified.
+
+**Note** :
+Make sure the ports are the same other than that, the connection will fail.
```

Comment:
**other than that** doesn't sound like the correct word choice here. I would change that to **otherwise**. **Other than that** can have different meanings than what you intend like introducing something. **Otherwise** in this context is more clear